python_src/go_jinja2/jinja2_renderer.py

```diff
@@ -44,6 +44,18 @@ def build_env(self):
         for e in self.opts.get("extensions", []):
             environment.add_extension(e)
 
+        for name, code in self.opts.get("filters", {}).items():
+            track = {}
+            exec(code, track)
+            f = None
+            for v in track.values():
+                if callable(v):
```

Comment:
is it somehow possible to check for the function name to equal the filter name? Just in case someone adds code with helper functions.